app/dashboard/views.py

```diff
@@ -2866,6 +2867,74 @@ def sync_web3(request):
     return JsonResponse(result, status=result['status'])
 
 
+# @require_POST
+# @csrf_exempt
+# @ratelimit(key='ip', rate='5/s', method=ratelimit.UNSAFE, block=True)
+def sync_etc(request):
+    """Sync up ETC chain to find transction status.
+
+    Returns:
+        JsonResponse: The JSON response following the web3 sync.
+
+    """
+
+    response = {
+        'status': '400',
+        'message': 'bad request'
+    }
+
+    # TODO: make into POST
+    txnid = request.GET.get('txnid', None)
+    bounty_id = request.GET.get('id', None)
+    network =  request.GET.get('network', 'mainnet')
+
+    # TODO: REMOVE
+    txnid = '0x30060f38c0e9e255061d1daf079d3707c640bfb540e207dbc6fc0e6e6d52ecd1'
+    bounty_id = 1
+
+    if not txnid:
+        response['message'] = 'error: transaction not provided'
+    elif not bounty_id:
+        response['message'] = 'error: issue url not provided'
+    elif network != 'mainnet':
+        response['message'] = 'error: etc syncs only on mainnet'
+    else:
+
+        # TODO: CHECK IF BOUNTY EXSISTS
+        # bounty = Bounty.object.get(pk=bounty_id)
+        # if not bounty:
+        #     response['message'] = f'error: bounty with key {bounty_id} does not exist'
+        # else:
+        #     print('bounty found') # wrap whole section below within else
+        retry = True
+        blockNumber = 0
+        confirmations = 0
+
+        # TODO: figure out retry / exponential backoff
+        while retry:
+            transaction = get_etc_txn_status(txnid, network)
+            if not transaction:
+                logging.error('blockscout failed')
+            elif transaction['has_mined']:
```

Comment:
Other options:
- we could off load the retry mechanism to the frontend if that makes sense !
- we could invoke the blockscout API from the frontend and then shoot a POST request to our backend once txn is mined